File: server/src/orders/models/feedback.model.ts
import {
  BelongsTo,
  Column,
  DataType,
  ForeignKey,
  Model,
  Table,
} from 'sequelize-typescript';

interface FeedbackCreationAttrs {
  name: string;
  email: string;
  feedback: string;
}

@Table({ tableName: 'feedbacks' })
export class Feedback extends Model<Feedback, FeedbackCreationAttrs> {
  @Column({
    type: DataType.INTEGER,
    autoIncrement: true,
    unique: true,
    primaryKey: true,
  })
  id: number;

  @Column({ type: DataType.STRING, allowNull: false, defaultValue: '' })
  name: string;

  @Column({ type: DataType.INTEGER, allowNull: false, defaultValue: 1 })
  email: string;

  @Column({ type: DataType.BOOLEAN, allowNull: false, defaultValue: false })
  feedback: string;
}

File: server/modules/orm/models/feedback.model.go
package model

import "time"

type Feedback struct {
	ID        uint64    `gorm:"primary_key, AUTO_INCREMENT"`
	Name      string    `json:"name"`
	Email     string    `json:"email"`
	Message   string    `json:"message"`
	CreatedAt time.Time `json:"created_at"`
}
